apps/website/src/content/docs/informationpanel.mdx

```diff
@@ -1,22 +1,65 @@
 ---
 title: Information panel
 description: A way to view additional attributes without leaving the list of items.
-propsPath: '@itwin/itwinui-react/esm/core/InformationPanel/InformationPanel.d.ts'
 thumbnail: InformationPanel
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='informationpanel' />
-
 <LiveExample src='InformationPanel.main.tsx'>
   <AllExamples.InformationPanelMainExample client:load />
 </LiveExample>
 
-The information panel is a pattern used primarily in tables. It allows viewing or editing additional details about selected item(s) without leaving the table.
+The Information Panel is designed for complex tables, offering a user-friendly way to explore content without overwhelming details or horizontal scrolling. It enhances data presentation in tables when needed.
+
+## Usage
+
+The Information Panel component is made up of customizable subcomponents. Here’s what the basic usage looks like:
+
+```tsx
+<InformationPanelWrapper>
+  <Table ... /> {/* component to overlay */}
+
+  <InformationPanel isOpen={true}>
+    <InformationPanelHeader onClose={() => {}}>
+      <Text variant='subheading'>InfoPanel heading</Text>
+    </InformationPanelHeader>
+
+    <InformationPanelBody>
+      <Text>Metadata</Text>
+      <InformationPanelContent ... />
+      <Text>Attributes</Text>
+      <InformationPanelContent ... />
+    </InformationPanelBody>
+
+  </InformationPanel>
+</InformationPanelWrapper>
+```
+
+### Overlay Component
+
+The `InformationPanel` component is designed to overlay content on the page. It is typically used to display additional information or actions without navigating away from the current view. By default, the panel is closed. It can be opened by setting `isOpen={true}`.
 
-The info panel is reserved for complex tables, because it allows the user to gradually explore the content without overwhelming them with too many details. It also helps avoiding the unpleasant horizontal scrolling when the table gets too wide. Not every table requires an information panel.
+### InformationPanelWrapper
+
+The `InformationPanelWrapper` is a container that is used to wrap and manage the placement of the `InformationPanel`. It helps in organizing and controlling the layout of the panel within the overall structure of the page.
+
+### InformationPanelContent
+
+The `InformationPanelContent` is used to display the content within `InformationPanel`. Inline/Block display style of the content can be set using the `displayStyle` prop.
+
+### onClose
+
+The `onClose` function is an event handler used to close or hide the `InformationPanel`. When this function is called, it sets the `openRowIndex` state to -1, effectively closing the panel.
```

Comment:
`openIndex` set ot `-1` is specific to this example only and no the component. So I believe the mention of "[openIndex is set to -1]" should be removed.

Actually, I think this entire heading can be removed since what `onClick` does can be implied from the name. Instead, you can briefly mention `onClick` near to where you mentioned `isOpen={true}` under "Overflow Component"

Reply:
removed